html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Parsing of meta refresh</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+iframe { display:none }
+</style>
+<body>
+<script>
+
+// failure to parse is []
+// success to parse is [time, url] where url is unresolved
+
+var tests_arr = [
+  {input: '', expected: []},
+  {input: '1', expected: [1, 'refresh.sub.html']},
+  {input: '1;', expected: [1, 'refresh.sub.html']},
+  {input: '1,', expected: [1, 'refresh.sub.html']},
+  {input: '1; url=foo', expected: [1, 'foo']},
```

Comment:
Maybe test more whitespace in between the punctuation and the url= (or lack of url=)?